dotcom-rendering/src/web/components/Card/components/LI.tsx

```diff
@@ -92,6 +98,7 @@ type Props = {
 	// Prevent the divider from spanning the LI's bottom padding. To be used when you know that the
 	// LI will have bottom padding, but won't have another card in the same container directly below it.
 	offsetBottomPaddingOnDivider?: boolean;
+	showOnMobile?: boolean;
```

Comment:
Added this in case in DCR we want to keep the functionality where [some containers show less cards in breakpoints below tablet](https://www.figma.com/file/sx2vMFHbL7SsUo0LcpsKNe/%E2%AC%A3--Front-container?node-id=0%3A1). Will remove if not. Do we want to keep this? @HarryFischer

Reply:
If we keep this, it will affect how we implement `Show More` functionality since in certain containers different cards will be shown in different breakpoints. My personal preference would be to not keep it as this will make things simpler.

Reply:
The requirement by digital editors is they want all containers, both fixed and dynamic, to keep the same count of cards on desktop vs mobile, so I am removing this.